CrudMicroservice.Domain/Services/EuroProtocolService.cs

```diff
@@ -29,19 +30,22 @@ public async Task<ResponseApiModel<HttpStatusCode>> RegistrationEuroProtocol(Eur
 
             await _euroProtocols.CreateAsync(euroProtocol);
 
-            return new ResponseApiModel<HttpStatusCode>(HttpStatusCode.OK, true, "Creating EuroProtocol is success!");
+            return new ResponseApiModel<HttpStatusCode>(HttpStatusCode.OK, true, Resources.ResourceManager.GetString("SuccessfulCreatingOfEuroProtocol"));
         }
 
         public async Task<ResponseApiModel<HttpStatusCode>> RegisterSideBEuroProtocol(SideRequestModel data)
         {
             var side = _mapper.Map<Side>(data);
             var update = Builders<EuroProtocol>.Update
-                .Set(c => c.IsClosed, true)
+                .Set(c => c.IsClosed, false)
```

Comment:
When we are register sideB,we also close euroProtocol,so isClosed=true

Reply:
fixed in web-editor